brianlong/www.validators.app
# 183911886 - fix validatorhistory orphans

#### What's this PR do?
PR fixes `validators_cli` method, which was creating validator_histories without related validator.
The orphan validator histories can be removed from db by the attached script.

#### How should this be manually tested?
- `rails test`
- use sql `"SELECT * FROM validator_histories vh WHERE (NOT EXISTS (SELECT * FROM validators v WHERE v.account = vh.account AND v.network = vh.network))"` in order to return validator_histories without any validators related and count the records
- run `rails r script/one_time_scripts/remove_orphan_validator_histories.rb`
- use again the above sql and check if there are no records returned.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183911886)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
